README.md

```diff
@@ -82,17 +87,36 @@ We prepared multiple environments:
    docker-compose pull && docker-compose up -d
    ```
 
-5. Go to [OnCall Plugin Configuration](http://localhost:3000/plugins/grafana-oncall-app), using log in credentials
-   as defined above: `admin`/`admin` (or find OnCall plugin in configuration->plugins) and connect OnCall _plugin_
-   with OnCall _backend_:
+5. Provision the plugin (If you run Grafana outside the included docker files, install the plugin before these steps):
```

Comment:
Maybe some mention about using the right Grafana admin credentials? (just in case not using default, similar as stated in the Troubleshooting section below)